polaris-core/src/main/java/org/apache/polaris/core/config/BehaviorChangeConfiguration.java

```diff
@@ -59,4 +59,13 @@ protected BehaviorChangeConfiguration(
           .description("Whether or not to use soft values in the entity cache")
           .defaultValue(false)
           .buildBehaviorChangeConfiguration();
+
+  public static final BehaviorChangeConfiguration<Boolean> TABLE_OPERATIONS_CACHE_METADATA =
+      PolarisConfiguration.<Boolean>builder()
+          .key("TABLE_OPERATIONS_CACHE_METADATA")
+          .description(
+              "If true, BasePolarisTableOperations should cache metadata that has been committed"
+                  + " which can reduce File IO")
+          .defaultValue(false)
```

Comment:
Disabling by default for the time being while I investigate some failing Iceberg tests